syscalls_ppc64le.go

```diff
@@ -0,0 +1,3 @@
+package ebpf
+
+const _BPFCall = 361
```

Comment:
Next task: add a builder for this ;P

Reply:
Haha at least we could test the online / available CPU parsing :P

```
arthur@steve ~/g/s/g/n/ebpf> cat /sys/devices/system/cpu/possible
0-143
arthur@steve ~/g/s/g/n/ebpf> cat /sys/devices/system/cpu/online
0-143
arthur@steve ~/g/s/g/n/ebpf> sudo ppc64_cpu --cores-on=1
arthur@steve ~/g/s/g/n/ebpf> cat /sys/devices/system/cpu/online
0-3
arthur@steve ~/g/s/g/n/ebpf> sudo ppc64_cpu --online-cores=6,8
arthur@steve ~/g/s/g/n/ebpf> cat /sys/devices/system/cpu/online
0-3,24-27,32-35
arthur@steve ~/g/s/g/n/ebpf> sudo ppc64_cpu --cores-on=all
arthur@steve ~/g/s/g/n/ebpf> cat /sys/devices/system/cpu/online
0-143
```